bytecode/concrete.py

```diff
@@ -116,23 +116,28 @@ def disassemble(cls, lineno, code, offset):
 
 class ConcreteBytecode(_bytecode.BaseBytecode, list):
 
-    def __init__(self):
+    def __init__(self, instructions=[]):
```

Comment:
Presumably you mean other `ConcreteBytecode` attributes. You make a good point there though, perhaps this PR should be scrapped.